helm/aws-load-balancer-controller/README.md

```diff
@@ -257,6 +257,8 @@ The default values set by the application itself can be confirmed [here](https:/
 | `podDisruptionBudget`                          | Limit the disruption for controller pods. Require at least 2 controller replicas and 3 worker nodes                                                                                                                                                                                                                                          | `{}`                                              |
 | `updateStrategy`                               | Defines the update strategy for the deployment                                                                                                                                                                                                                                                                                               | `{}`                                              |
 | `enableCertManager`                            | If enabled, cert-manager issues the webhook certificates instead of the helm template, requires cert-manager and it's CRDs to be installed                                                                                                                                                                                                   | `false`                                           |
+| `certManager.duration`                            | Set the expiry duration for the webhook certificates. Used only when `enableCertManager: true`                                                                                                                                                                                                  | `""`                                           |
```

Comment:
This doesn't look correct to me, if `enableCertManager` is set to `true`, the webhook is issued by the cert-manager instead of supplied by helm chart. so the value you try to add in helm chart won't take effect.